prac_05/wimbledon.py

```diff
@@ -0,0 +1,23 @@
+import csv
+filename = "wimbledon.csv"
+print("Wimbledon Champions:")
+champions_to_victories = {}
+champions = []
+champion_countries = []
+longest_name = 0
+with open(filename, "r") as in_file:
+    reader = csv.reader(in_file)
+    next(reader)
```

Comment:
A comment explaining the purpose of this line (removing headers) might be helpful?